File: 3726-RemoveZerosInDecimalRepresentation/3726-RemoveZerosInDecimalRepresentation.py
# Last updated: 9/12/2026, 7:18:58 AM
class Solution(object):
    def removeZeros(self, n):
        x=""
        s=str(n)
        for i in range(len(s)):
            if s[i]!='0':
                x+=s[i]
        return int(x)
        

File: 3726-RemoveZerosInDecimalRepresentation/3726-RemoveZerosInDecimalRepresentation.java
// Last updated: 9/12/2026, 7:19:05 AM
class Solution {
    public long removeZeros(long n) {
        String s="";
        while(n!=0){
            long a=n%10;
            if(a!=0){
                s=String.valueOf(a)+s;
            }
            n/=10;
        }
        return Long.parseLong(s);
    }
}